config/symbols.hd.dra.txt

```diff
@@ -1,11 +1,7 @@
 D_800AC910 = 0x800AC8BC;
 D_800AC914 = 0x800AC8D0;
 D_800AC934 = 0x800AC9FC;
-D_800ACE88 = 0x800ACE88; // ignore:True
```

Comment:
Revert these changes to fix the build, the "remove unused symbols" script is buggy